packages/jest-cli/src/SearchSource.js

```diff
@@ -174,7 +175,7 @@ class SearchSource {
     };
   }
 
-  findChangedTests(): Promise<SearchResult> {
+  findChangedTests(lastCommit: boolean): Promise<SearchResult> {
```

Comment:
can we make this an object called `options` with a `lastCommit` field? That way we can extend this API later.